File: corelib/src/main/java/org/apache/cordova/domen/KwkResponseNew.java
package org.apache.cordova.domen;

import androidx.annotation.Keep;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class KwkResponseNew {

    public KwkResponseNew() {} //We need empty constructor!

    @SerializedName("client_id")
    @Expose
    public String clientId;
    @SerializedName("session_id")
    @Expose
    public String sessionId;
    @SerializedName("offer_id")
    @Expose
    public Long offerId;
    @SerializedName("response")
    @Expose
    public String response;
    @SerializedName("message")
    @Expose
    public String message;

    @Keep
    @SerializedName("product")
    @Expose
    public String product;

    @Override
    public String toString() {
        return "KwkResponseNew{" +
                "clientId='" + clientId + '\'' +
                ", sessionId='" + sessionId + '\'' +
                ", offerId=" + offerId +
                ", response='" + response + '\'' +
                ", message='" + message + '\'' +
                ", product='" + product + '\'' +
                '}';
    }
}

File: kwk/corelib/src/main/java/org/apache/cordova/domen/KwkResponseNew.kt
package org.apache.cordova.domen

import androidx.annotation.Keep
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class KwkResponseNew  //We need empty constructor!
{
    @JvmField
    @SerializedName("client_id")
    @Expose
    var clientId: String? = null

    @SerializedName("session_id")
    @Expose
    var sessionId: String? = null

    @SerializedName("offer_id")
    @Expose
    var offerId: Long? = null

    @JvmField
    @SerializedName("response")
    @Expose
    var response: String? = null

    @SerializedName("message")
    @Expose
    var message: String? = null

    @Keep
    @SerializedName("product")
    @Expose
    var product: String? = null

    override fun toString(): String {
        return "KwkResponseNew{" +
                "clientId='" + clientId + '\'' +
                ", sessionId='" + sessionId + '\'' +
                ", offerId=" + offerId +
                ", response='" + response + '\'' +
                ", message='" + message + '\'' +
                ", product='" + product + '\'' +
                '}'
    }
}
